.ci/scripts/local.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-kustomize build config/local | kubectl apply -f -
+kustomize build config/local | kubectl apply --server-side=true -f -
```

Comment:
The `kubectl apply` command adds a `.metadata.annotation` (_kubectl.kubernetes.io/last-applied-configuration_) into the resource.

Because of the number of fields of our CRD, when we try to run the command we are getting the error:
```
The CustomResourceDefinition "pulps.repo-manager.pulpproject.org" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
```

Using the [server-side apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/) this issue does not happen.